packages/nc-gui/components/smartsheet/Gallery.vue

```diff
@@ -159,6 +159,12 @@ onMounted(async () => {
 
 // provide view data reload hook as fallback to row data reload
 provide(ReloadRowDataHookInj, reloadViewDataHook)
+
+watch(view, (nextView) => {
+  if (nextView?.type === ViewTypes.FORM) {
+    reloadEventHook.trigger()
+  }
+})
```

Comment:
if the view type is form, then the Gallery component will not be mounted anyway, so this seems redundant.